source/Halibut.Tests/BaseTest.cs

```diff
@@ -15,15 +15,16 @@ public class BaseTest
         public CancellationToken CancellationToken { get; private set; }
         public ILogger Logger { get; private set; } = null!;
 
-
         [SetUp]
         public void SetUp()
         {
-            traceLogFileLogger = new TraceLogFileLogger();
+            traceLogFileLogger = new TraceLogFileLogger(SerilogLoggerBuilder.CurrentTestHash());
             Logger = new SerilogLoggerBuilder()
                 .SetTraceLogFileLogger(traceLogFileLogger)
                 .Build()
                 .ForContext(GetType());
+            
+            Logger.Information("Trace log file {LogFile}", traceLogFileLogger.logFilePath);
```

Comment:
Thanks for this bit, I've got some WIP code to associate the TeamCity artifact with the test run so the trace log file can be accessed in-line in the build log, but there's still some TeamCity black magic I need to figure out to get it to work.

This will help in the meantime!

Reply:
Ah nice!

I also like this message for local development, since when the test is run the first line is "here is more info if you need it"

Reply:
Yep, it was on my to-do list for that exact reason, but got distracted by the TeamCity black magic and it didn't make it to a PR.